virocon/plotting.py

```diff
@@ -18,6 +39,26 @@ def get_default_model_description(n_dim):
 
 
 def plot_marginal_quantiles(model, sample, model_desc=None, axes=None):
+    """
+    Method to plot marignal quantiles of a distribution. 
+    
+    Plots the fitted marginal distribution versus a dataset in a quantile-
+    quantile (QQ) plot.
+    
+    Parameters
+    ----------
+    model :  MultivariateModel
+        The model to be used to plot the marignal quantiles.
+    sample : ndarray of floats
+        The environmental data sample that should be plotted against the fit.
+    model_desc: dict, optional
+        Generated model description. Defaults to None.
```

Comment:
The description of the model. If None (the default) a generic model description will be used. 

TODO link to example